src/include/daos_fs.h

```diff
@@ -69,6 +70,19 @@ typedef struct {
 	daos_range_t	       *iod_rgs;
 } dfs_iod_t;
 
+/**
+ * Get the DFS superblock D-Key and A-Keys
+ *
+ * \param[out] dkey DFS superblock D-Key
+ * \param[out] iods DFS superblock A-keys
+ * \param[out] dfs_entry_size number of superblock A-keys
+ *
+ * \return              0 on success, errno code on failure.
+ */
+int
+get_sb_layout(daos_key_t *dkey, daos_iod_t *iods[], int *akey_count,
```

Comment:
any DFS API function should have the dfs_* prefix.
But why do you think this should be an API function?
this seems too specific to your use case and needs not to be exposed as an API function.

Reply:
Done, function was removed from the public API